superset-frontend/src/features/alerts/AlertReportModal.tsx

```diff
@@ -1011,7 +1015,15 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
       !isEditMode &&
       (!currentAlert || currentAlert.id || (isHidden && show))
     ) {
-      setCurrentAlert({ ...defaultAlert });
+      setCurrentAlert({
+        ...defaultAlert,
+        owners: [
+          {
+            value: currentUser.userId,
+            label: `${currentUser.firstName} ${currentUser.lastName}`,
+          },
+        ],
```

Comment:
Note: I was going to propose breaking this out into a util, but it turns out we have a lot of different `User` types that are very different (sometimes `id` us used, other times it's `userId`, and sometimes it's `firstName`, other times `first_name` etc). So best leave this as-is and do a separate PR to clean up all user types in one go.

Reply:
Ack.